docs/content/docs/ops/batch/recovery_from_jobMaster_failover.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Recovery from JobMaster failover"
+weight: 4
+type: docs
+aliases:
+
+- /docs/ops/batch/recovery_from_jobMaster_failover.html
+- /docs/ops/batch/recovery_from_jobMaster_failover
+
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Batch jobs recovery from JobMaster failover
+
+## Background
+
+Currently, when Flink’s JobMaster crashes resulting in termination, it will cause the following two situations:
+
+- If the job has not enabled high availability (HA), the job will fail.
+- If the job has enabled HA, in the case of a streaming job, it will resume from the last successful checkpoint
+  after the JobMaster failover. However, for a batch job, which does not have checkpoints, it will have to start
```

Comment:
If the job has enabled HA, -> If HA is enabled, a job master failover would happen and the job will be restarted.

in the case of a streaming job, it will resume from the last successful checkpoint after the JobMaster failover. -> For streaming jobs, it can resume from the last successful checkpoint.